docs/streams/quickstart.html

```diff
@@ -140,25 +130,36 @@
     --topic streams-file-input
 </pre>
 
+The created topic can be described with the same <b>kafka-topics</b>> tool:
 
 <pre class="brush: bash;">
-&gt; bin/kafka-console-producer.sh --broker-list localhost:9092 --topic streams-file-input < file-input.txt
+&gt; bin/kafka-topics.sh --zookeeper localhost:2181 --describe
+
+Topic:streams-file-input	PartitionCount:1	ReplicationFactor:1	Configs:
+    Topic: streams-file-input	Partition: 0	Leader: 0	Replicas: 0	Isr: 0
+</pre>
+
+And start the console producer that can then write to this topic line-by-line,
```

Comment:
Suggestion:

> Now we can start the console producer to write some input data to this topic.  Enter a line of text and then hit `<RETURN>`.  This will send a new message to the input topic, where the message key is `null` and the message value is the text line that you just entered.  (In practice, input data for applications will typically be streaming continuously into Kafka, rather than being manually entered as we do in this quickstart.)